superbench/benchmarks/micro_benchmarks/cuda_memory_bw_performance.py

```diff
@@ -108,23 +71,16 @@ def _process_raw_result(self, cmd_idx, raw_output):
         self._result.add_raw_data('raw_output_' + self._args.mem_type[cmd_idx], raw_output)
 
         mem_bw = -1
-        metric = ''
         valid = True
         content = raw_output.splitlines()
         try:
-            for index, line in enumerate(content):
-                if 'H2D' in line:
-                    metric = 'H2D_Mem_BW'
-                elif 'D2H' in line:
-                    metric = 'D2H_Mem_BW'
-                elif 'D2D' in line:
-                    metric = 'D2D_Mem_BW'
-                else:
-                    continue
-                line = line.split(',')[1]
-                value = re.search(r'(\d+.\d+)', line)
-                if value:
-                    mem_bw = max(mem_bw, float(value.group(0)))
+            metric = self._metrics[self._mem_types.index(self._args.mem_type[cmd_idx])]
+            for line in content:
+                if 'D2H' in line or 'H2D' in line or 'D2D' in line:
```

Comment:
`mem_bw` should be just one type of result. Do you think we should keep a `parse_logline_map` dict, and we check the log line as follows:
```
parse_logline = parse_logline_map[cmd_idx]
if parse_log_line in line:
```

Reply:
I try to update a version, please check if it's ok

Reply:
It works for me.